micromasters/urls.py

```diff
@@ -27,8 +28,8 @@
     url(r'^api/v0/dashboard/$', UserDashboard.as_view(), name='dashboard_api'),
     url(r'^api/v0/search/(?P<elastic_url>.*)', ElasticProxyView.as_view(), name='search_api'),
     url(r'^api/v0/checkout/$', CheckoutView.as_view(), name='checkout'),
+    url(r'^api/v0/mail$', MailView.as_view(), name='mail_api'),
```

Comment:
here's why i didn't add the slash:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/django/core/handlers/base.py", line 123, in get_response
    response = middleware_method(request)
  File "/usr/local/lib/python3.5/site-packages/django/middleware/common.py", line 62, in process_request
    path = self.get_full_path_with_slash(request)
  File "/usr/local/lib/python3.5/site-packages/django/middleware/common.py", line 100, in get_full_path_with_slash
    'url': request.get_host() + new_path,
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to 192.168.99.100:8079/api/v0/mail/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
```

i'm happy to add the `APPEND_SLASH` setting if that's what we want to go with. i went with the path of least resistance for this PR